org.freedesktop.Sdk.json.in

```diff
@@ -41,6 +41,13 @@
             "version": "3.22",
             "download-if": "active-gtk-theme"
         },
+        "org.gtk.Gtk3module" : {
+            "directory": "lib/gtk3-modules",
+            "merge-dirs": "modules",
+            "no-autodownload": true,
+            "subdirectories": true,
+            "version": "3.0"
```

Comment:
It is tied to the runtime so `1.6` makes sense here.

Reply:
It definately doesn't make sense. ABI is tied to a specific runtime, you can't expect random code build against e.g. freedesktop 1.6 to work against any other runtime.

Reply:
I mean, the Gtk+ 3 ABI is stable on its *own*, but the rest of the ABI is not. For example, there is no guarantee that a Gtk3 module built on debian jessie works on the Gtk3 build of fedora 29, which is similar to two runtimes both shipping gtk3.